### PR TITLE
FAQ - update 'rooted_devices' entry

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Remove "also" from english heading
- Change answer to "yes"

### CODE REVIEW

**Comment by fynngodau** [comment]
Saying "yes" here is, in total, a good change. Users should not be discouraged from using the app for no reason. However, the article should mention that unlocking the bootloader or similar modifications may cause SafteyNet checks to fail, meaning that users cannot participate in PPDD features. (https://github.com/corona-warn-app/cwa-wishlist/issues/356)

File: src/data/faq_de.json [hunk 1]
```diff
@@ -405,7 +405,8 @@
                         "anchor": "rooted_devices",
                         "active": true,
                         "textblock": [
-                            "Nein, die App wird auf gerooteten Geräten nicht unterstützt."
+                            "Ja, das ist möglich. Die App wird auf gerooteten Geräten jedoch nicht offiziell unterstützt.",
+                            "Zurzeit führen das Entsperren des Bootloaders or ähnliche Modifikationen (wie die Installation von microG) dazu, dass die Benutzung der optionalen Datenspende- und Befragungsfunktionen ohne Warnung fehlschlägt. De Grund dafür ist, dass diese Funktionen einen Integritätscheck über SafetyNet benötigen. Wir prüfen derzeit alternative Lösungen zur jetzigen Implementierung, siehe <a href='https://github.com/corona-warn-app/cwa-wishlist/issues/356' target='_blank' rel='noopener noreferrer'>corona-warn-app/cwa-wishlist#356</a>."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:409]: "or ähnliche" => "oder ähnliche"
"De Grund" => "Der Grund"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I noticed two typos in the German text.

File: src/data/faq_de.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq_de.json:409]: thx all fixed

**Comment by heinezen** [comment]
@fynngodau I added your remarks. Pls double-check if what I wrote is correct :)

File: src/data/faq_de.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq_de.json:409]: @heinezen 
I'm not seeing the changes to 
"or ähnliche" => "oder ähnliche"
"De Grund" => "Der Grund"

**Comment by fynngodau** [comment]
@heinezen Installing microG is technically not really similar to unlocking the bootloader, but both will cause SafteyNet to fail in many cases, so it can be left as is. :+1:

File: src/data/faq_de.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq_de.json]: fixed

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
LGTM, but let's get internal approval for this one.

**Comment by MikeMcC399** [comment]
@dsarkar 
What is the intention for this PR about Android rooted devices? It has not made any progress for several months.

According to https://github.com/corona-warn-app/cwa-app-android/pull/4135 CWA 2.12 will introduce a root check:

"Die App prüft bei einem Neustart, ob auf Ihrem Smartphone Administrator-Berechtigungen (auch bekannt also Root-Berechtigungen) aktiv sind, und zeigt gegebenenfalls einen Sicherheitshinweis an."

so it would be good to plan an update of the article in time for the release of 2.12 (the release after the next one - we are currently at 2.10).

**Comment by dsarkar** [comment]
DE
image

EN
image

**Comment by dsarkar** [comment]
@Ein-Tim Yep, should have done to branch darkar-release-2-12

**Comment by Ein-Tim** [comment]
@dsarkar I guess we're close to the 2.12 release? If yes, we can probably leave it like this for now...

**Comment by dsarkar** [comment]
PR https://github.com/corona-warn-app/cwa-website/pull/1072

**Pull request merged**